index.bs

```diff
@@ -1213,7 +1214,7 @@ The [=native WebGL framebuffer resolution=] is determined by running the followi
 
   1. Let |session| be the target {{XRSession}}.
   1. If |session|'s {{XRSession/mode}} value is not <code>"inline"</code>, set the [=native WebGL framebuffer resolution=] to the resolution required to have a 1:1 ratio between the pixels of a framebuffer large enough to contain all of the session's {{XRView}}s and the physical screen pixels in the area of the display under the highest magnification and abort these steps. If no method exists to determine the native resolution as described, the [=recommended WebGL framebuffer resolution=] MAY be used.
-  1. If |session|'s {{XRSession/mode}} value is <code>"inline"</code>, set the [=native WebGL framebuffer resolution=] to the size of the |session|'s {{XRSession/outputContext}}'s {{XRPresentationContext/canvas}} in physical display pixels and reevaluate these steps every time the size of the canvas changes.
+  1. If |session|'s {{XRSession/mode}} value is <code>"inline"</code>, set the [=native WebGL framebuffer resolution=] to the size of the |session|'s {{XRSession/renderState}}'s {{XRRenderState/outputContext}}'s {{XRPresentationContext/canvas}} in physical display pixels and reevaluate these steps every time the size of the canvas changes.
```

Comment:
Given that the output context can be changed, where are we specifying "native-webgl-framebuffer-resolution" be invoked?

Reply:
Added a line to the `native-webgl-framebuffer-resolution` algorithm indicating it should be re-evaluated when the `outputContext` changes. As for application of that algorithm, that'll probably be something that's detailed whenever we get around to detailing the compositor/swap chain more rigorously.

Reply:
(small favor to ask... can you hold off on squashing commits until the final merge?  It would have been a lot easier to review this fix if I could look at the additive commit ^_^)